types/netmap/netmap.go

```diff
@@ -177,6 +177,13 @@ func (nm *NetworkMap) MagicDNSSuffix() string {
 	return MagicDNSSuffixOfNodeName(nm.Name)
 }
 
+func (nm *NetworkMap) TailnetName() string {
```

Comment:
nit: doc

Reply:
I also would probably name this exactly what the netmap calls it. We've got a couple of "TailnetNames" going around, I think it's better to be explicit, even if we expose this as "Tailnet name" in UI